AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -1051,6 +1049,12 @@ - (void)transitionLayoutWithSizeRange:(ASSizeRange)constrainedSize
       
       // Kick off animating the layout transition
       [self animateLayoutTransition:_pendingLayoutTransitionContext];
+        
+      // Mark transaction as finished
+      [self _finishOrCancelTransition];
+      
+      // As we prevented to do a layout pass while the layout transition was in flight
+      [self setNeedsLayout];
```

Comment:
Do we necessarily want to do this every time? Assuming it results in the same final layout, the UIKit layout pass that comes as a result of this, won't interfere with the inflight animations?